src/frontend/types/task_types.ts

```diff
@@ -0,0 +1,18 @@
+import * as tasksConstants from '../constants/tasks_constants'
+
+export interface ReceiveTasks {
```

Comment:
I don't think you need to export interfaces, but I was seeing a compilation error if I pulled them out. ¯\_(ツ)_/¯ I don't export anything from `interfaces.ts` and that seems to work fine. not sure why this is different.

Reply:
I figured this out! Interfaces in a `*.d.ts` file are magically picked up. Interfaces in a regular file (that is, inline with things) have to be imported and exported. So for global things (say, a `Task`) we'll have an `interfaces.d.ts` somewhere but otherwise will do it as you wrote it.